models/data.js

```diff
@@ -47,81 +44,80 @@ function reader_norm(out, path, callback) {
     } catch (e) {
       console.error(
         "Exception while parsing file",
-        path + file,
+        pathname + file,
         ": ",
         e.message
       );
       return;
     }
-
     // TODO: There should be a try/catch around this
-    _.each(numbers, function (number_data, number_key) {
-      float_key = parseFloat(number_key, 10);
-      if (isNaN(float_key)) {
-        console.log(
-          "Skipping invaid number_key",
-          number_key,
-          "in file",
-          path + file
-        );
-        return;
-      }
-
-      // TODO: handle this during normalization
-      if (!number_data || number_data.length === 0) {
-        // console.log('Skipping empty number_data for float_key', float_key, 'in file', path + file);
-        return;
-      }
-
-      if (!(float_key in out)) {
-        out[float_key] = [];
-      }
-      var o = out[float_key];
-
-      _.each(number_data, function (element) {
-        if (!element.text || !element.text.length) {
+    try {
+      _.each(numbers, function (number_data, number_key) {
+        float_key = parseFloat(number_key, 10);
+        if (isNaN(float_key)) {
           console.log(
-            "Skipping empty file (element.text is falsey)",
-            path + file
+            "Skipping invaid number_key",
+            number_key,
+            "in file",
+            pathname + file
           );
           return;
         }
-        if (callback) {
-          element = callback(element);
-        }
-        if (!element) {
+
+        // TODO: handle this during normalization
+        if (!number_data || number_data.length === 0) {
+          // console.log('Skipping empty number_data for float_key', float_key, 'in file', pathname + file);
           return;
         }
-        if (!element.manual) {
-          if (element.text.length < 20 || element.text.length > 150) {
+
+        if (!(float_key in out)) {
+          out[float_key] = [];
+        }
+        var o = out[float_key];
+        _.each(number_data, function (element) {
+          if (!element.text || !element.text.length) {
+            console.log(
+              "Skipping empty file (element.text is falsey)",
+              pathname + file
+            );
             return;
           }
+          if (callback) {
+            element = callback(element);
+          }
+          if (!element) {
+            return;
+          }
+          if (!element.manual) {
+            if (element.text.length < 20 || element.text.length > 150) {
```

Comment:
Can you replace these [magic numbers](https://en.wikipedia.org/wiki/Magic_number_(programming)#Unnamed_numerical_constants) with named variables?